rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "1.70.0"
```

Comment:
any reason for 1.70.0 in particular ?

Reply:
evolved over time ;) let me try to bring it down to 1.69 again (which is the required version for solana 1.16.13)